### PR TITLE
Layout changes for the collector and storage using zipkin layout factory

### PR DESCRIPTION
**Pull request by zeagord**
Changed the the kafka10 collector build layout based on Zipkin-Layout-Factory

### CODE REVIEW

File: zipkin-autoconfigure/collector-kafka10/pom.xml [hunk 1]
```diff
@@ -43,22 +43,23 @@
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
         <version>${spring-boot.version}</version>
-        <executions>
-          <execution>
-            <goals>
-              <goal>repackage</goal>
```
Comment (codefromthecrypt) [inline zipkin-autoconfigure/collector-kafka10/pom.xml:49]: is repackage redundant? or is there another reason why this isn't needed anymore?

Comment (zeagord) [reply zipkin-autoconfigure/collector-kafka10/pom.xml:49]: Yeah it is redundant and IMHO repackage is not needed. If we need to we have to provide the launcher in the layout factory as optional. Is there a situation where we need to run this as a standalone jar?

Comment (codefromthecrypt) [reply zipkin-autoconfigure/collector-kafka10/pom.xml:49]: nope.. just asking!

**Comment by codefromthecrypt** [comment]
thanks tons

**Pull request merged**